TACC-Cloud/hazmapper
# Task/WG-10: Add projects and user queries.

## Overview: ##
Adds basic project and user queries for react.

## PR Status: ##


## Related Jira tickets: ##

* [WG-10](https://jira.tacc.utexas.edu/browse/WG-10)

## Summary of Changes: ##
Added basic query to userinfo and projects to production geoapi.
Also added slice for projects and modified `authSlice` to include user information.
Added prettier in npm script.

## Testing Steps: ##
1. NOTE: Testing is set with production geoapi.
2. Open page (http://localhost:4200)
3. Ensure in chrome networks, you see the projects and userinfo responses.

## UI Photos:

## Notes: ##
I've tried setting locally with the jwt.. For some reason I get a CORS error for endpoints that should have that covered.
